visualizer/actionButton.js

```diff
@@ -154,6 +253,24 @@
     buttonWrapper.hidden = false;
   }
 
+  function handleContextMenuOnAction(nameContainer) {
```

Comment:
Nice! All your DOM handling code is very nice and clear, good names and useful comments.